python/ql/src/experimental/semmle/python/Concepts.qll

```diff
@@ -38,3 +38,60 @@ module SystemCommandExecution {
     abstract DataFlow::Node getCommand();
   }
 }
+
+/**
+ * A data-flow node that decodes data from a binary or textual format. This
+ * is intended to include deserialization, unmarshalling, decoding, unpickling,
+ * unzipping, decrypting, parsing etc.
```

Comment:
To me, `unzip` means that you extract files from a zip-archive and put them on disk. I'm guessing decompressing is more appropriate here? (but do correct me if I'm wrong)

```suggestion
 * decompressing, decrypting, parsing etc.
```